clients/src/main/resources/common/message/JoinGroupRequest.json

```diff
@@ -30,7 +30,9 @@
   // Version 6 is the first flexible version.
   //
   // Version 7 is the same as version 6.
-  "validVersions": "0-7",
+  //
+  // Version 8 adds the Reason field (KIP-800).
+  "validVersions": "0-8",
```

Comment:
We need to bump the request version as well.

Reply:
bumped the response version